resources/views/pages/printerFriendlyDoorSign.blade.php

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE HTML>
+<html lang="en">
+	<head>
+		@if ( config('app.google_site_id') )
```

Comment:
Let's remove this guy per our discussion since we don't want google analytics on this guy